gr-blocks/lib/packed_to_unpacked_impl.h

```diff
@@ -24,7 +24,7 @@ class packed_to_unpacked_impl : public packed_to_unpacked<T>
     const unsigned int d_bits_per_chunk;
     const endianness_t d_endianness;
     unsigned int d_index;
-    static constexpr unsigned int d_bits_per_type = sizeof(T) * 8;
+    static constexpr unsigned int d_bits_per_type() { return (sizeof(T) * 8); }
```

Comment:
```suggestion
    static constexpr unsigned int get_bits_per_type() { return (sizeof(T) * 8); }
```

The `d_` is to indicate non-static class members.